dev-util/smartgit/smartgit-6.5.6.ebuild

```diff
@@ -23,7 +23,7 @@ RESTRICT="fetch"
 RDEPEND=">=virtual/jre-1.7:1.7"
```

Comment:
Is `git` or `hg` needed a runtime? Or add a use flag to pull them in like `app-portage/layman` does.